### ISSUE TITLE
dbus-adc: puts services on D-Bus without DeviceInstance

### ISSUE THREAD

**Issue by mpvader** [victronenergy/venus#255]
```
root@beaglebone:~# dbus -y com.victronenergy.tank.builtin_adc4_di0 / GetValue
value = {'Capacity': 0.20000000298023224,
 'Connected': [],
 'DeviceInstance': [],
 'FluidType': 0,
 'Level': [],
 'Mgmt/Connection': 'Tank Level sensor input 1',
 'Mgmt/ProcessName': 'dbus-adc',
 'Mgmt/ProcessVersion': '1.16',
 'ProductId': [],
 'ProductName': [],
 'Remaining': [],
 'Standard': 0,
 'Status': 1,
 'analogpinFunc': 1}
```

Same for a temperature sensor:
```
root@beaglebone:~# dbus -y com.victronenergy.temperature.builtin_adc3_di1 / GetValue
value = {'Connected': [],
 'DeviceInstance': [],
 'Mgmt/Connection': 'Temperature sensor input 2',
 'Mgmt/ProcessName': 'dbus-adc',
 'Mgmt/ProcessVersion': '1.16',
 'Offset': 0,
 'ProductId': [],
 'ProductName': [],
 'Scale': 1.0,
 'Status': 1,
 'Temperature': [],
 'TemperatureType': 0,
 'analogpinFunc': 1}
```

**Comment by mpvader** [comment]
velib_python is the service that fails on this:

```
2018-01-29 15:19:12.574569500 MainThread-vrmlogger: /opt/victronenergy/vrmlogger/vrmlogger.py v2.91 is starting up
2018-01-29 15:19:12.579098500 MainThread-vrmlogger: Loglevel set to INFO
2018-01-29 15:19:12.605298500 MainThread-vedbus: registered ourselves on D-Bus as com.victronenergy.logger
2018-01-29 15:19:12.632749500 MainThread-dbusmonitor: ===== Search on dbus for services that we will monitor starting... =====
2018-01-29 15:19:12.639165500 MainThread-dbusmonitor: Found: com.victronenergy.system, scanning and storing items
2018-01-29 15:19:12.647422500 MainThread-dbusmonitor:        com.victronenergy.system has device instance 0
2018-01-29 15:19:12.759082500 MainThread-dbusmonitor: Found: com.victronenergy.settings, scanning and storing items
2018-01-29 15:19:12.768227500 MainThread-dbusmonitor:        com.victronenergy.settings has device instance 0
2018-01-29 15:19:12.860709500 MainThread-dbusmonitor: Found: com.victronenergy.tank.builtin_adc4_di0, scanning and storing items
2018-01-29 15:19:12.868513500 Traceback (most recent call last):
2018-01-29 15:19:12.872040500   File "/opt/victronenergy/vrmlogger/vrmlogger.py", line 585, in <module>
2018-01-29 15:19:12.872050500     main()
2018-01-29 15:19:12.872052500   File "/opt/victronenergy/vrmlogger/vrmlogger.py", line 476, in main
2018-01-29 15:19:12.876092500     vebusDeviceInstance0=True
2018-01-29 15:19:12.876100500   File "/opt/victronenergy/vrmlogger/./ext/velib_python/dbusmonitor.py", line 108, in __init__
2018-01-29 15:19:12.876105500     self.scan_dbus_service(serviceName)
2018-01-29 15:19:12.876108500   File "/opt/victronenergy/vrmlogger/./ext/velib_python/dbusmonitor.py", line 176, in scan_dbus_service
2018-01-29 15:19:12.876113500     device_instance = 0 if device_instance is None else int(device_instance)
2018-01-29 15:19:12.876117500 TypeError: int() argument must be a string or a number, not 'dbus.Array'
2018-01-29 15:19:14.485069500 MainThread-vrmlogger: /opt/victronenergy/vrmlogger/vrmlogger.py v2.91 is starting up
```

**Comment by mpvader** [comment]
@izak it does seem that between v2.11 (dbus-adc 1.14) and v2.12~26 (dbus-adc 1.16) something changed wrt the DeviceInstance:

```
root@beaglebone:~# dbus -y com.victronenergy.tank.builtin_adc4_di0 / GetValue
value = {'Capacity': 0.20000000298023224,
 'Connected': 1,
 'DeviceInstance': 20,
 'FluidType': 0,
 'Level': [],
 'Mgmt/Connection': 'Tank Level sensor input 1',
 'Mgmt/ProcessName': 'dbus-adc',
 'Mgmt/ProcessVersion': '1.14',
 'ProductId': 41313,
 'ProductName': 'Generic Tank Input',
 'Remaining': [],
 'Standard': 0,
 'Status': 1,
 'analogpinFunc': 1}
```

**Comment by mpvader** [comment]
Fixed in v2.12~31
